src/profiles.js

```diff
@@ -26,24 +28,20 @@ async function httpGetAsync(url) {
 
 function pkDataToProfile(data) {
   let profile = {
-    name: data.member.name,
-    color: '#' + data.member.color,
-    tag: data.system.tag,
+    name: data.member.display_name || data.member.name,
+    color: '#' + (data.member.color || 'ffffff'),
```

Comment:
Defaulting to white here seems incorrect to me. Doesn't that break the existing color logic of memberColour and tagColour in ColorMessageHeader.js